### PR TITLE
Fix color code in Makefile

### LINKED ISSUE #78: Color codes do not work in Makefile
In the Makefile, the color codes do not seem to work and do not output any color.

When running `make test`, the following is outputted.
```
\033[32;01m==> Cleaning bytecode ...\033[0m
\033[32;01m==> Linting code ...\033[0m
```

### PR DESCRIPTION
**Pull request by Unviray**
fix #78

### CODE REVIEW

**Comment by pavdmyt** [comment]
Hi @Unviray 

This doesn't work for me (macOS Mojave 10.14.6):

```
$ which shell
shell not found
```

**Comment by Unviray** [comment]
Does the `make` command produce an error?

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-OK_COLOR=\033[32;01m
-NO_COLOR=\033[0m
+OK_COLOR := $(shell tput -Txterm setaf 2)
```
Comment (pavdmyt) [inline Makefile:1]: Does `-Txterm` is a necessary parameter? What happens if I'm using non-xterm terminal?

**Comment by pavdmyt** [comment]
@Unviray 

I've read some docs and figured out how `shell` + `tput` works here. 
Thanks for the PR!

**Comment by Unviray** [comment]
Travis-ci build failed. I don't know why .

File: Makefile (hunk 1 above)
Comment (pavdmyt) [reply Makefile:1]: @Unviray please check this comment ^^

**Pull request merged**